app/controllers/transcriptions_controller.rb

```diff
@@ -114,6 +141,21 @@ def approving?
     update_attrs["status"] == "approved"
   end
 
+  def fresh?
+    # the 'If-Unmodified-Since' datetime will be sent over by client with ISO 8601 format, 3 digits of fractional seconds
+    @transcription.updated_at.iso8601(3) == request.headers['If-Unmodified-Since']
```

Comment:
Checkout how rails does this for conditional get requests, https://github.com/rails/rails/blob/4257ea756b0a5fbec4719aebf5ff846105833e73/actionpack/lib/action_dispatch/http/cache.rb#L12

I'm pretty sure you are onto this but the trick is to ensure the serialized format going out matches what comes back in, https://github.com/rails/rails/blob/f33d52c95217212cbacc8d5e44b5a8e3cdc6f5b3/actionpack/lib/action_controller/metal/conditional_get.rb#L115